docs/Guides/CreateCustomer.md

```diff
@@ -0,0 +1,139 @@
+---
+tags: [Customers]
+---
+# Sync Contact Information From Your System to Vendasta
+
+> This guide relies on "Proposed" endpoints and is likely to change until all features have been made generally available.
+
+Vendasta allows you to sync contact information from your CRM or POS system onto our platform. Doing so will allow you to use features in our platform like SMS messaging through Vendasta's Inbox or [requesting business review through Customer Voice](./CustomerVoice/SendReviewRequest.md).
```

Comment:
We should address the difference between sales contacts for partners and customers for SMBs. (And clean up the terminology)

Reply:
Yeah, that's an issue. Are we safe sticking with the model name "Customers" or should we opt for a more descriptive one?